actions/pack_action.go

```diff
@@ -34,8 +37,19 @@ func (pf *PackAction) Run(context *debos.DebosContext) error {
 	pf.LogStart()
 	outfile := path.Join(context.Artifactdir, pf.File)
 
+	var tarOpt = "cf"
+	tarOpts := map[string]string{
+		"gz":    "z",
+		"bzip2": "j",
+		"xz":    "J",
+	}
+
+	if tarOptCompression := tarOpts[pf.Compression]; len(tarOptCompression) > 0 {
```

Comment:
you are silently allowing user to choose parameters which may not exist. It makes sense to check the value in the `Verify` function of the action. that way the recipe will fail before its even ran.

Reply:
I've added a check in Verify for early failure.